lib/package/menu.coffee

```diff
@@ -36,11 +36,18 @@ module.exports =
         label: 'Working Directory'
         submenu: [
           {label: 'Current File\'s Folder', command: 'julia-client:work-in-current-folder'}
-          {label: 'Current Project\'s Folder', command: 'julia-client:work-in-project-folder'}
+          {label: 'Current Project\'s Folders', command: 'julia-client:work-in-project-folder'}
```

Comment:
```suggestion
          {label: 'Select Project Folder', command: 'julia-client:work-in-project-folder'}
```
since this opens the project selector.